lib/src/domain/common/openalias_record.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:basic_utils/basic_utils.dart';
+
+class OpenaliasRecord {
```

Comment:
Let make this class as dump data holder which will just store name and monero address. So for get it user will be needed to call static method which will return filled instance of `OpenaliasRecord` if will find the record.